tools/hasher/pom.xml

```diff
@@ -38,14 +38,6 @@
         <dependency>
             <groupId>org.apache.shiro</groupId>
             <artifactId>shiro-core</artifactId>
-            <exclusions>
-                <!-- BeanUtils is for ini config - this simple command line program
-                     does not init a shiro environment, so we don't need it: -->
-                <exclusion>
-                    <groupId>commons-beanutils</groupId>
-                    <artifactId>commons-beanutils</artifactId>
-                </exclusion>
-            </exclusions>
         </dependency>
```

Comment:
Do you really want to introduce one extra dependency for just one invocation?

Reply:
I was able to make reflection a one-liner, so that works just as well as `PropertyUtils`
Thanks!